patches/chrome-browser-ui-views-frame-browser_window_factory.cc.patch

```diff
@@ -0,0 +1,22 @@
+diff --git a/chrome/browser/ui/views/frame/browser_window_factory.cc b/chrome/browser/ui/views/frame/browser_window_factory.cc
+index a82f7fe13172c7aca4cbd2dfb75c22295f43ac95..2214617485607dcbe546e786902a5c3ae0ba9b2e 100644
+--- a/chrome/browser/ui/views/frame/browser_window_factory.cc
++++ b/chrome/browser/ui/views/frame/browser_window_factory.cc
+@@ -4,7 +4,7 @@
+ 
+ #include "build/build_config.h"
+ #include "chrome/browser/ui/views/frame/browser_frame.h"
+-#include "chrome/browser/ui/views/frame/browser_view.h"
++#include "brave/browser/ui/views/frame/brave_browser_view.h"
+ #include "chrome/browser/ui/views/frame/native_browser_frame_factory.h"
+ #include "chrome/browser/ui/views_mode_controller.h"
+ #include "chrome/grit/chromium_strings.h"
+@@ -24,7 +24,7 @@ BrowserWindow* BrowserWindow::CreateBrowserWindow(Browser* browser,
+ #endif
+   // Create the view and the frame. The frame will attach itself via the view
+   // so we don't need to do anything with the pointer.
+-  BrowserView* view = new BrowserView();
++  BrowserView* view = new BraveBrowserView();
```

Comment:
This can be done via chromium_src override.
So create a file in src/brave/chrome/browser/ui/views/frame/browser_window_factory.cc

At the top of the file include `#include "chrome/browser/ui/views/frame/browser_view.h"` s that it doesn't get renamed too.

Then

#define BrowserView BraveBrowserView
Then `#include "../../../../../../../chrome/browser/ui/views/frame/browser_window_factory.cc"`